client.go

```diff
@@ -155,6 +155,12 @@ type Config struct {
 	// and the client. If this is set to -1, the client will not attempt to
 	// send client -> server PING requests.
 	PingDelay time.Duration
+	// PingTimeout specifies the duration at which girc will assume
+	// that the connection to the server has been lost if no PONG
+	// message has been received in reply to an outstanding PING.
+	//
+	// The configured PingTimeout must be larger than PingDelay.
```

Comment:
See **Notes to reviewer** above.

Reply:
PingTimeout should be the time between a `PING` and no `PONG`, and so `PingDelay` should be irrelevant, as that's between two `PING`s.